src/index.js

```diff
@@ -0,0 +1,127 @@
+; (() => {
+
+    const appElement = document.createElement("div")
+    appElement.id = "app"
+    document.body.appendChild(appElement)
+
+    // HEADER
+    const headerElement = document.createElement("header")
```

Comment:
There is no sense to create header, div and p elements in js as they as static. What you should really do:
- create them in html
- add id to p element
- control value of p inside js